css/styles.css

```diff
@@ -103,11 +110,89 @@ header .menu {
   width: 100%;
 }
 
-ul,
-li {
-  list-style-type: none;
+header .my-logo,
+header .menu {
+  display: block;
+  width: 100%;
+}
+
+.menu-content {
+  width: 100%;
+  height: 100%;
+  flex-direction: column;
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  background: #1b31f0a2;
+  backdrop-filter: blur(4px);
 }
 
+.menu-content .x-menu {
+  display: flex;
+  align-self: flex-end;
+  padding: 3.75rem 2.375rem;
+}
+
+.menu-content .menu-list-item {
+  margin-top: 2.5rem;
+  padding: 0 1.75rem;
+}
+
+.menu-content a {
+  font-size: 2rem;
+  line-height: 4rem;
+  font-weight: 600;
```

Comment:
None of your `font-weight` rules are working. That is most likely caused by not importing the right weights from Google Fonts. Please refer to [this video](https://www.youtube.com/watch?v=mRoBBOTYNz4&t=184s) so you can fix this issue.

<img width="561" alt="Screen Shot 2022-07-08 at 7 45 46" src="https://user-images.githubusercontent.com/82426682/177994671-38fc803c-f54f-4127-89c8-27fd2b5023f0.png">

Reply:
@vechicin thank you for this video, I have imported the font in all it format and applied the suggested changes